### DISCUSSION TITLE
Disable user sign up

### DISCUSSION THREAD

**Discussion by iamwillbar** [nocodb/nocodb#785] [Q&A]
Is it possible to disable the self-serve user sign up?

**Comment by wingkwong** [comment]
You can disable it by adding the environment variable ``NC_INVITE_ONLY_SIGNUP``. Value can be anything other than an empty string.

**Reply by ovizii** [reply]
that is great for stopping signups but how do I invite people? Say I want to test nocodb with a friend, how do I invite him? Of course I can work around that by enabling signups, having him sign up, then disable signups again.

Also, there doesn't seem to be a user management interface to administer my nocodb installation, all I can find is the user management interface for users of a specific project, please correct me if I missed something.

**Reply by mingfang** [reply]
You can send invite using the `Share` button on the upper right corner.

**Reply by ovizii** [reply]
> You can send invite using the `Share` button on the upper right corner.

Yes, correct, but that is inviting someone to collaborate with the current project I assumed seeing that the "Share" button is only available when I am inside a project.

I was looking for a way to invite someone to use my nocodb instance on their own and create their own projects, just like what the signup option seems to do.

**Reply by o1lab** [reply]
If the invited user is ```creator``` or ```owner``` - they will be able to create their own projects as well.

**Reply by ovizii** [reply]
Thanks, maybe I'm having a blonde moment or just not getting it. Let me phrase my question differently:

After logging in I see my projects:
![image](https://user-images.githubusercontent.com/837902/148585335-9c5e6f77-4e88-4982-9cce-01cc03feab42.png)

Say I want to give my brother access to let him test nocodb too, can he not have his own account on my own instance without having access to my projects but rather create his own? 

@iamwillbar - sincere apologies for high-jacking your thread with this related question, if the next reply doesn't help me figure it out I'll open my own thread :-)

**Reply by o1lab** [reply]
Ah I see your point :) thank you for taking a moment and putting this. We are reconsider the ACL part. And will look into this.

At the moment you can invite and once in - he can be deleted from the project :) as a workaround
